lib/cequel/record/associations.rb

```diff
@@ -108,14 +108,16 @@ def belongs_to(name, options = {})
                  "belongs_to association must be declared before declaring " \
                  "key(s)"
           end
-          
+
           key_options = options.extract!(:partition)
+          foreign_key_parts = [options.fetch(:foreign_key, [])].flatten
```

Comment:
Oh and for the record i was never that happy with the `flatten` :) Probably wouldn't have passed review where I work either! LOL